src/test/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+jdbc.driverName=org.h2.Driver
+jdbc.url=jdbc:h2:file:./src/test/resources/h2/sample
+jdbc.username=sa
+jdbc.password=
+
+marklogic.host=localhost
+marklogic.port=8000
+marklogic.username=admin
+marklogic.password=admin
+marklogic.database=msb-content
+
+marklogic.jobrepo.host=localhost
```

Comment:
Let's not use a jobrepo by default, I want this to be as simple as possible - just focused on getting data from an RDBMS into ML. A jobrepo is one more thing for the user to understand right away.